nasa-veda-platform/scientific-computing/github-authentication.qmd

```diff
@@ -0,0 +1,54 @@
+---
+title: How to push to GitHub from the VEDA JupyterHub
+---
+
+This is a quick how-to guide for pushing to GitHub from the VEDA JupyterHub. There are several methods for GitHub Authentication, which are detailed in [GitHub's documentation](https://docs.github.com/en/authentication). While some of those are applicable across all compute environments (e.g., personal access tokens), we recommend using [`gh-scoped-creds`](https://github.com/jupyterhub/gh-scoped-creds) as a fine tuned GitHub authenticatin method designed for shared computing environments like HPCs and JupyterHubs.
+
+## User authentication via veda-hub-github-scoped-creds app
+
+### Copy notebook for running `gh-scoped-creds`
+
+
+Copy the `/home/jovyan/veda-docs/notebooks/templates/template-github-auth.ipynb` notebook to your personal home directory on the Hub as `github-auth.ipynb`.
+
+::: {.callout-tip}
+You only need to do this step once.
+:::
+
+### Authenticate to GitHub
+
+::: {.callout-tip}
+You need to do this step each time you create a new JupyterHub instance.
```

Comment:
Create or Boot (start)? I feel like the word could be confusing, but maybe I'm overthinking.